tests/cypress/support/commands.js

```diff
@@ -227,6 +227,21 @@ Cypress.Commands.add(
                 if (!withHash && 'adminId' in queryParams) {
                     // In legacy drawing with adminId the layer is not added to the layers parameter
                     target += 1
+                    // When handling a legacy parameter, the bod Layer Id might add extra
+                    // layers, thus the need to check if those extra layers have been added
```

Comment:
you might want to rephrase this and change the "BOD layer ID" to "Feature preselection", it might make it clearer what this is all about